www/docs/contribute.rst

```diff
@@ -1,17 +1,43 @@
 contribute
-===========
+------------
 
-There is lots of fun work to be done!
+We're so glad you're interested in helping make rovercode even better! Whether you've got years of experience or you're pretty new to coding, you can make great contributions, and we are here to help and support you.
 
-Head on over to `the rovercode github <https://github.com/aninternetof/rovercode>`_.
-We use ZenHub to improve GitHub's agile management, so `install it <https://www.zenhub.com/>`_,
-then visit the `boards` tab to find a fun card in the backlog. Or `submit a new
-card <https://github.com/aninternetof/rovercode/issues/new>`_ for a bug or cool
-new feature idea.
+You can get in touch with the folks at rovercode in lots of ways:
 
-And remember, you can do all these same things for
-`rovercode-web <https://github.com/aninternetof/rovercode-web>`_.
+- Pop into `our chatroom <http://chat.rovercode.com>`_.
 
-Chat with us on `the rovercode Slack channel <http://chat.rovercode.com>`_.
+- Add a comment to the ticket you're working on.
 
-Follow the `the code of conduct <code_of_conduct.html>`_.
+- Email developers@rovercode.com .
+
+And if you ever need to, we'd be happy to set up a time to video chat and screen-share!
+
+What is rovercode?
+====================
+rovercode is an open-source educational robotics platform. Create code that listens to the rover's sensors and controls its motors, then watch your code in action as the rover sets out across your classroom! We enable students to hit the ground coding.
+Read more at `rovercode.com <https://rovercode.com>`_.
+
+How to Get Started
+=====================
+- Join our `our chat <http://chat.rovercode.com>`_ and say hello in any of the rooms!
```

Comment:
This line has `our` twice

Reply:
Sorry -- not sure how that happened again!